test/python_tests.txt

```diff
@@ -392,7 +394,7 @@ test_signal
 test_site subprocess
 test_slice
 test_smtpd
-test_smtplib
+test_smtplib  baad ioctl syscall 21537
```

Comment:
These are new failures, but I think it best to address them as a follow-on to this PR.